core/connection.js

```diff
@@ -410,18 +411,19 @@ Blockly.Connection.singleConnection_ = function(block, orphanBlock) {
  *     of blocks, or null.
  * @package
  */
-Blockly.Connection.lastConnectionInRow = function(startBlock, orphanBlock) {
-  var newBlock = startBlock;
-  var connection;
-  while ((connection = Blockly.Connection.singleConnection_(
-      /** @type {!Blockly.Block} */ (newBlock), orphanBlock))) {
-    newBlock = connection.targetBlock();
-    if (!newBlock || newBlock.isShadow()) {
-      return connection;
-    }
-  }
-  return null;
-};
+Blockly.Connection.getPlaceForOrphanedOutput =
```

Comment:
Why this rename? I prefer the name to include "Connection" in it rather than "Place", but I do like adding the "get" in front.
What about `getLastConnectionInRow`?

Reply:
I'm totally cool with swithcing `Place` to `Connection` but I don't think anything with `Last` or `Row` is an accurate representation of what this function is doing.

For example, pretend you did not know the logic this function is running. Would you expect something called `getLastConnectionInRow` to return the highlighted connection in these stacks of blocks:
![middle](https://user-images.githubusercontent.com/25440652/118892042-24beaf80-b8b5-11eb-8bf8-2893e147e639.png)
![end](https://user-images.githubusercontent.com/25440652/118892046-26887300-b8b5-11eb-8ef1-798f01fe792d.png)

Or no connection for this stack of blocks:
![none](https://user-images.githubusercontent.com/25440652/118892075-31db9e80-b8b5-11eb-839b-df7aae6b9e5b.png)

Imo it's not a general use function. It's only used for finding a good connection to reattach an orphan to, so it should be named as such.

This is opposed to `lastConnectionInStack`, which ignores statement inputs and connection checks. This makes it consistant and general purpose, so I think the name is great in that situation.